exampleSite/config.toml

```diff
@@ -47,7 +47,7 @@ disqusShortname = "yourdiscussshortname"
     maxSeeAlsoItems = 5
 
     # Enable Twemoji
-    enableTwemoji = true
+    enableTwemoji = false
```

Comment:
This change will make the [twemoji example article](https://hugo-coder.netlify.app/posts/twemoji-support/) stop working.
So, no need to changes here.